cmd/auth/main.go

```diff
@@ -52,6 +54,7 @@ const (
 	defKetoHost      = "mainflux-keto"
 	defKetoWritePort = "4467"
 	defKetoReadPort  = "4466"
+	defLoginDuration = "50"
```

Comment:
Make default `10h` to be aligned with the old version.

Reply:
Changed